### DISCUSSION TITLE
Date filter by Month [Apr 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5517] [Discord Threads]
> 🙋‍♂️ **Damir**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### this helps?  
![image0](https://cdn.discordapp.com/attachments/1084827895614165073/1084853327801630831/Screenshot_2023-03-13_at_8.29.05_PM.png)  
#  
> 🙋‍♂️ **sukhan**  
#### I wonder why my filter doesn't look the same - the filter fields are different...  
![image0](https://cdn.discordapp.com/attachments/1084827895614165073/1084916415523475606/filter_date.png)  
#  
> 🙋‍♂️ **Damir**  
#### I don't have those options on my instance...  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### My bad. These options are in develop & will be available from next release.  
#  
> 🙋‍♂️ **Damir**  
#### Thanks 👍  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1084827895614165073)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
